image/image.go

```diff
@@ -55,33 +55,39 @@ const (
  * Image trailer TLV types.
  */
 const (
-	IMAGE_TLV_KEYHASH   = 0x01
-	IMAGE_TLV_SHA256    = 0x10
-	IMAGE_TLV_RSA2048   = 0x20
-	IMAGE_TLV_ECDSA224  = 0x21
-	IMAGE_TLV_ECDSA256  = 0x22
-	IMAGE_TLV_RSA3072   = 0x23
-	IMAGE_TLV_ED25519   = 0x24
-	IMAGE_TLV_ENC_RSA   = 0x30
-	IMAGE_TLV_ENC_KEK   = 0x31
-	IMAGE_TLV_ENC_EC256 = 0x32
-	IMAGE_TLV_AES_NONCE = 0x50
-	IMAGE_TLV_SECRET_ID = 0x60
+	IMAGE_TLV_KEYHASH          = 0x01
+	IMAGE_TLV_SHA256           = 0x10
+	IMAGE_TLV_RSA2048          = 0x20
+	IMAGE_TLV_ECDSA224         = 0x21
+	IMAGE_TLV_ECDSA256         = 0x22
+	IMAGE_TLV_RSA3072          = 0x23
+	IMAGE_TLV_ED25519          = 0x24
+	IMAGE_TLV_ENC_RSA          = 0x30
+	IMAGE_TLV_ENC_KEK          = 0x31
+	IMAGE_TLV_ENC_EC256        = 0x32
+	IMAGE_TLV_AES_NONCE_LEGACY = 0x50
+	IMAGE_TLV_SECRET_ID_LEGACY = 0x60
+	IMAGE_TLV_AES_NONCE        = 0xa1
```

Comment:
Should we document why ranges 0x70 - 0xa0 not used? As it is used by MCUBoot?

Reply:
The range is supposed to match mcuboot.

Reply:
mcuboot drives the fixed TLV numbers.